### PR TITLE
Upgrade administration frontend dependencies

### PR DESCRIPTION
**Pull request by maxammann**
Sadly I failed upgrading react-scripts. It still has rough edges and fixes are not released yet

### CODE REVIEW

**Comment by maxammann** [comment]
I also had to keep graphql on version 15.8 because the "apollo" tooling package is not yet compatible with the new version

File: administration/src/App.tsx [hunk 1]
```diff
@@ -51,19 +51,20 @@ const App = () => <AuthProvider>
...
+                                                                              icon="form"
+                                                                              text="Eingehende Anträge"/></NavLink>
+                                        <NavLink to={"/eak-generation"}><Button icon="people"
+                                                                                text="Karten erstellen"/></NavLink>
+                                    </div>}/>
+                                    <Route path={"/applications"}
+                                           element={<ApplicationsController token={authData.token}/>}/>
+                                    <Route path={"/eak-generation"} element={<GenerationController/>}/>
+                                </Routes>
                             </Main>
```
Comment (Schedulaar) [inline administration/src/App.tsx:68]: Why did you remove the `exact` flags for Route `/applications` and `/`?

File: administration/src/components/generation/GenerationForm.tsx [hunk 2]
```diff
@@ -72,10 +72,10 @@ const GenerationForm = (props: Props) => {
 
     const allCardsValid = cardCreationModels.reduce((acc, model) => acc && isValid(model), true)
 
+    usePrompt("Falls Sie fortfahren, werden alle Eingaben verworfen.", cardCreationModels.length !== 0);
+    
     return (
         <>
-            <Prompt message={"Falls Sie fortfahren, werden alle Eingaben verworfen."}
-                    when={cardCreationModels.length !== 0}/>
             <ButtonBar stickyTop={0}>
```
Comment (Schedulaar) [inline administration/src/components/generation/GenerationForm.tsx:79]: Why are react routing libraries so shitty and unstable...

**Review by Schedulaar — APPROVED** [review]

File: administration/src/App.tsx (hunk 1 above)
Comment (maxammann) [reply administration/src/App.tsx:68]: its no longer supported, should still work, at least from my tests

File: administration/src/components/generation/GenerationForm.tsx (hunk 2 above)
Comment (maxammann) [reply administration/src/components/generation/GenerationForm.tsx:79]: My words...

Loving Flutter/Angular for that.
